### PR TITLE
Search component

### PR DESCRIPTION
**Pull request by RodrigoRG1921**
Initial search component view

### CODE REVIEW

**Comment by ravermz** [comment]
Please fix the merge conflicts and then I will review it

**Comment by RodrigoRG1921** [comment]
Fixed!

File: src/components/SearchComponent.tsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+import {TextInput, IconButton, Surface} from '@react-native-material/core'
+import Icon from 'react-native-vector-icons/MaterialIcons'
+
+const SearchComponent = () => {
+  return (
+    <View style={{flexDirection: 'row'}}>
+      <TextInput
+        variant="outlined"
+        label="Search"
+        style={{margin: 10, width: '75%'}}
```
Comment (ravermz) [inline src/components/SearchComponent.tsx:12]: a good practice is set 100% in width property and the reduce the space in the parent component (view container)

**Review by ravermz — APPROVED** [review]

File: src/components/SearchComponent.tsx (hunk 1 above)
Comment (RodrigoRG1921) [reply src/components/SearchComponent.tsx:12]: thanks sir

**Pull request merged**